code/game/objects/items/storage/firstaid.dm

```diff
@@ -517,3 +517,111 @@
 /obj/item/storage/pill_bottle/paxpsych/PopulateContents()
 	for(var/i in 1 to 5)
 		new /obj/item/reagent_containers/pill/paxpsych(src)
+
+/obj/item/storage/organbox
+	name = "organ transport box"
+	desc = "An advanced box with an cooling mechanism that uses cryostylane or other cold reagents to keep the organs or bodyparts inside preserved."
+	icon_state = "organbox"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	throw_speed = 3
+	throw_range = 7
+	custom_premium_price = 1100
+	/// var to prevent it freezing the same things over and over
+	var/cooling = FALSE
+
+/obj/item/storage/organbox/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_w_class = WEIGHT_CLASS_BULKY /// you have to remove it from your bag before opening it but I think that's fine

```

Comment:
minor thing, i think this will allow carrying egg sacs and big plasma vessels in your backpack which wasnt possible previously

Reply:
those items only appear when a dead xeno is butchered next to a surgery table for organ implanting and during meaty ores,  i dont see any relevant change of gameplay with the combination of these items